fe/fe-core/src/main/java/org/apache/doris/catalog/HiveMetaStoreClientHelper.java

```diff
@@ -192,6 +194,9 @@ public static String getHiveDataFiles(HiveTable hiveTable, ExprNodeGenericFuncDe
                     // path = "/path/to/partition/file_name"
                     // eg: /home/work/dev/hive/apache-hive-2.3.7-bin/data/warehouse/dae.db/customer/state=CA/city=SanJose/000000_0
                     String path = fileStatus.getPath().toUri().getPath();
+                    if (onS3) {
```

Comment:
Add an example in comment to explain the transformation.

Reply:
Added